### DISCUSSION TITLE
react-datepicker mode onBlur

### DISCUSSION THREAD

**Discussion by dawinValenzuela** [react-hook-form/react-hook-form#6378] [Q&A]
Hello everyone, first to all Thank you for your hel. 
I have an integration with react-hook-form and react-datepicker. I'm also using Controller. here in this example
https://codesandbox.io/s/react-hook-form-react-datepicker-2wiv5?file=/src/App.js
the input is empty, so if I click, I saw the calendar put lets say I don't select any date. so, the react-hook-form show me the error. that's right so I click again and select the date.. the input set the value but the error still there.. maybe I'm missing something. 

thank you all!

**Comment by bluebill1049** [comment]
just tried your example, looks like its working. can you share the steps to produce the problem?

**Reply by dawinValenzuela** [reply]
hey @bluebill1049 sure. do not click the button. 
1- click inside the input. 
2- don’t select any date. just click outside the calendat.
3 - this action will trigger the error. 
4- click again the input and select a date.
5 - this set the value in the input but the error still there.. 

should no be there when the value is set it right?

thank you.
